src/gei/Program.cs

```diff
@@ -24,7 +25,19 @@ public static async Task Main(string[] args)
                 .AddSingleton<AdoApiFactory>()
                 .AddTransient<ITargetGithubApiFactory>(sp => sp.GetRequiredService<GithubApiFactory>())
                 .AddTransient<ISourceGithubApiFactory>(sp => sp.GetRequiredService<GithubApiFactory>())
-                .AddHttpClient();
+                .AddHttpClient("NoSSL")
+                .ConfigurePrimaryHttpMessageHandler(() => new HttpClientHandler()
+                {
+                    AllowAutoRedirect = false,
+                    CheckCertificateRevocationList = false,
+                    ServerCertificateCustomValidationCallback = delegate { return true; }
+                })
+                .Services
```

Comment:
I'm surprised this even compiles without parens after it. 😮

Reply:
why? it's legit code, it's just a property 😃

Reply:
Turns out to be a property!